clang/runtime/dpct-rt/include/dpct/graph.hpp

```diff
@@ -22,5 +22,31 @@ typedef sycl::ext::oneapi::experimental::command_graph<
     sycl::ext::oneapi::experimental::graph_state::executable>
     *command_graph_exec_ptr;
 
+static inline std::unordered_map<sycl::queue *, command_graph_ptr>
+    queue_graph_map;
```

Comment:
If there is more than one translation unit, each translation unit will have its own `queue_graph_map` variable. The operation in translation unit 1 is invisible from translation unit 2.
I think queue_graph_map should be a singleton (like dpct::mem_mgr) rather than a static inline variable.

Reply:
Done.